src/server.js

```diff
@@ -9,6 +9,13 @@ import socialStrategy from './middlewares/socialStrategy';
 import docs from '../swagger.json';
 import router from './routers/index';
 import passportConfig from './middlewares/localStrategy';
+import { sequelize } from './models';
+
+const SequelizeStore = require('connect-session-sequelize')(session.Store);
+
+const Store = new SequelizeStore({
```

Comment:
Well done @chikeozulumba. This is a trivial change I'm suggesting: Is there a reason why you are capitalizing `Store`, instead of `store`?. To make it clear to someone that `Store` is referring to an instance of `SequelizeStore`, you might want to change it to `store`